presto-docs/src/main/sphinx/optimizer/push-down.rst

```diff
@@ -0,0 +1,140 @@
+=========
+Push down
+=========
+
+Presto can push down processing queries, or segments of queries, down to the
+connected data source. This means that a specific function, or other operation,
+is passed through to the underlying database for processing.
+
+The results of this push down can include the following benefits:
+
+* improved overall query performance
+* reduced network traffic between Presto and the database
+* reduced load on the database
+
+Support for push down is specific to each connector and the relevant underlying
+database or storage system.
+
+Every Presto aggregate function is eligible for push down, as long as it
+operates on types supported by the given connector.
+
+Analysis and Confirmation
+-------------------------
+
+Push down depends on a number of factors:
+
+* generic support for push down in the connector
+* function or operation specific support for push down in the connector
```

Comment:
I'm not sure what the difference between these two items are. I think we can remove this section, since attempts to rephrase basicly result in "things can be pushed down if the connector supports it".

Reply:
From what I know there are various aspects that need to be in place.

* Presto framework needs to support push down of that function
* connector needs to support that push down
* underlying data source needs to actually have a function equivalent

So its not just that the connector needs to support it ..

Reply:
The list has these various cases imho .. but I am happy to improve with any suggestions